SystemInfo/Value.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * File containing the Value class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+/**
+ * System information Value.
+ */
+abstract class Value extends ValueObject
```

Comment:
Implemented as interface now, still empty. Now it's up to the implementors to extend ValueObject, not sure I like this: You could make a Value that isn't a ValueObject.

Naming issues still open.